app.js

```diff
@@ -61,6 +64,12 @@ function newUser(event) {
       userExists = true;
```

Comment:
how is this for loop better than the each loop they had? Wouldn't you want to use `forEach`? #functionalprogramming #ftw